eclipse-theia/theia
# ui/ux: fix bug induced by PR #12604

#### What it does

This PR fixes a bug that was induced in #12604 where the styling for input fields would not be displayed properly. An example would be the input field appearing when renaming a file. This was caused by `overflow: hidden` being applied on all text-inputs. Now, the `overflow: hidden` will only be applied to the `vsx-extension-search-bar` which requires it.

Before:
![image](https://github.com/eclipse-theia/theia/assets/86936229/c9d495a3-286a-49f3-9880-1f7f967ca806)

After:
![image](https://github.com/eclipse-theia/theia/assets/86936229/e3a2349b-e8a4-4663-89c5-33eb4b8de19c)


#### How to test


1. Start Theia, open the `Extensions` or the `Source Control` views
2. Resize the view, making it smaller
3. The input field should be responsive, with the text overflow hidden by an ellipsis
4. Try renaming a file, the input field should be styled correctly

#### Review checklist


#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
